src/wireviz/DataClasses.py

```diff
@@ -7,6 +7,9 @@
 from wireviz.wv_helper import int2tuple, aspect_ratio
 from wireviz import wv_colors
 
+ConnectorMultiplier = str  # = Literal['pincount', 'populated']
+CableMultiplier = str  # = Literal['wirecount', 'terminations', 'length', 'total_length']
+
```

Comment:
Any reason the `Literal`s are commented out?

Reply:
from @kvid's comment https://github.com/formatc1702/WireViz/pull/115#discussion_r497943873
> Note that the Literal type specifier is in comments to avoid requiring Python 3.8.

I have checked my 2.7 install and literals aernt supported, doing some searching I cant find any reference to this method of specifying literals so not sure if this is a standard implementation method or not.

Reply:
Understood. Let's keep it like this then, perhaps add a comment behind the comment explaining this.

Reply:
New in version 3.8 is specified by https://docs.python.org/3/library/typing.html#typing.Literal